catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/operations/ResourceOperationsOptionsTest.java

```diff
@@ -0,0 +1,365 @@
+
+    QueryResponse queryResponseMock;
+
+    QueryOperations queryOperationsMock;
+
+    boolean fanoutEnabled;
+
+    URI testUri;
+
+    Map<String, Serializable> requestPropertiesMock;
+
+    MetacardImpl metacard;
+
+    QueryRequest queryRequestMock;
+
+    @Before
+    public void setUp() throws Exception {
+        setUpFrameworkProperties();
+        setUpResourceOperationsAndMocks();
+    }
+
+    private void setUpResourceOperationsAndMocks() throws URISyntaxException {
+        queryOperationsMock = mock(QueryOperations.class);
+        queryResponseMock = mock(QueryResponse.class);
+        queryRequestMock = mock(QueryRequest.class);
+        requestPropertiesMock = mock(Map.class);
+
+        testUri = new URI("bobUri", "test", "fragment");
+
+        metacard = new MetacardImpl();
+        metacard.setId("Bobbert");
+        metacard.setTitle("Bobbert's Title");
+
+        resourceOperations = new ResourceOperations(frameworkProperties,
+                queryOperationsMock,
+                opsSecuritySupport);
+    }
+
+    private void setUpFrameworkProperties() {
+        frameworkProperties = new FrameworkProperties();
+        frameworkProperties.setAccessPlugins(new ArrayList<>());
+        frameworkProperties.setPolicyPlugins(new ArrayList<>());
+        frameworkProperties.setSourcePoller(mockPoller);
+        frameworkProperties.setCatalogProviders(Collections.singletonList((CatalogProvider) provider));
+        frameworkProperties.setPostResource(mockPostResourcePlugins);
+        frameworkProperties.setFederationStrategy(mockFederationStrategy);
+        frameworkProperties.setFilterBuilder(new GeotoolsFilterBuilder());
+        frameworkProperties.setPreIngest(new ArrayList<>());
+        frameworkProperties.setPostIngest(postIngestPlugins);
+        frameworkProperties.setPreQuery(new ArrayList<>());
+        frameworkProperties.setPostQuery(new ArrayList<>());
+        frameworkProperties.setPreResource(new ArrayList<>());
+        frameworkProperties.setPostResource(new ArrayList<>());
+        frameworkProperties.setQueryResponsePostProcessor(mock(QueryResponsePostProcessor.class));
+        frameworkProperties.setStorageProviders(Collections.singletonList(storageProvider));
+        frameworkProperties.setMimeTypeMapper(new MimeTypeMapperImpl(Collections.singletonList(
+                mimeTypeResolver)));
+        frameworkProperties.setMimeTypeToTransformerMapper(mimeTypeToTransformerMapper);
+        frameworkProperties.setValidationQueryFactory(new ValidationQueryFactory(new GeotoolsFilterAdapterImpl(),
+                new GeotoolsFilterBuilder()));
+    }
+
+    @Test
+    public void testGetEnterpriseResourceOptions() throws Exception {
+
+        Result resultMock = mock(Result.class);
+
+        ArrayList<Result> mockResultList = new ArrayList<>();
+        mockResultList.add(resultMock);
+
+        when(queryOperationsMock.query(any(), any(), anyBoolean(), anyBoolean())).thenReturn(
```

Comment:
To echo a comment Eric left on my unit test PR, we should avoid using `any()` as much as possible. Instead, specify types with `any(ClassName.class)`.